package/crds/mysql.sql.crossplane.io_users.yaml

```diff
@@ -70,6 +70,22 @@ spec:
                     - name
                     - namespace
                     type: object
+                  resourceOptions:
+                    description: ResourceOptions sets account specific resource limits. See https://dev.mysql.com/doc/refman/8.0/en/user-resources.html
+                    properties:
+                      maxConnectionsPerHour:
+                        description: MaxConnectionsPerHour sets the number of times an account can connect to the server per hour
+                        type: integer
+                      maxQueriesPerHour:
+                        description: MaxQueriesPerHour sets the number of queries an account can issue per hour
+                        type: integer
+                      maxUpdatesPerHour:
+                        description: MaxUpdatesPerHour sets the number of updates an account can issue per hour
+                        type: integer
+                      maxUserConnections:
+                        description: MaxUserConnections sets The number of simultaneous connections to the server by an account
+                        type: integer
+                    type: object
```

Comment:
Not to sound like a broken record... but should there be defaults here to avoid new comers from being stuck wondering what values are "ok"?

Reply:
Optional but you are essentially enforcing a default in the actual code of the provider, right? Or did I read that wrong?

Reply:
Can you point out where that is? I'm probably overlooking something.

Reply:
Sorry man, I failed to read for detail. I remembered seeing some specific values when I scanned the diff but now as I look back I see they're from the example yaml. *face palm*